payment/order_test.go

```diff
@@ -1,28 +1,114 @@
 package payment
 
 import (
+	"encoding/hex"
 	"testing"
 
+	"github.com/asaskevich/govalidator"
+	macarooncmd "github.com/brave-intl/bat-go/cmd/macaroon"
+	"github.com/brave-intl/bat-go/utils/cryptography"
 	"github.com/stretchr/testify/suite"
 )
 
 type OrderTestSuite struct {
+	service *Service
 	suite.Suite
 }
 
 func TestOrderTestSuite(t *testing.T) {
 	suite.Run(t, new(OrderTestSuite))
 }
 
+func (suite *OrderTestSuite) SetupSuite() {
+	govalidator.SetFieldsRequiredByDefault(true)
+	pg, err := NewPostgres("", false, "")
+	suite.Require().NoError(err, "Failed to get postgres conn")
+
+	m, err := pg.NewMigrate()
+	suite.Require().NoError(err, "Failed to create migrate instance")
+
+	ver, dirty, _ := m.Version()
+	if dirty {
+		suite.Require().NoError(m.Force(int(ver)))
+	}
+	/*
+		if ver > 0 {
+			suite.Require().NoError(m.Down(), "Failed to migrate down cleanly")
+		}
+	*/
+
+	EncryptionKey = "MTIzNDU2Nzg5MDEyMzQ1Njc4OTAxMjM0"
+	InitEncryptionKeys()
+
+	suite.Require().NoError(pg.Migrate(), "Failed to fully migrate")
+	suite.service = &Service{
+		Datastore: pg,
+	}
+}
+
+func (suite *OrderTestSuite) TearDownTest() {
+	//	suite.CleanDB()
+}
+
+/*
```

Comment:
Do we need this?

Reply:
resolved: https://github.com/brave-intl/bat-go/pull/816/commits/ce69229aff8904385f828224ffc65ec983526b7b